src/patients/appointments/AppointmentsList.tsx

```diff
@@ -54,6 +54,20 @@ const AppointmentsList = (props: Props) => {
 
   return (
     <Container>
+      <div className="row">
+        <div className="col-md-12 d-flex justify-content-end">
+          <Button
+            outlined
+            color="success"
+            icon="add"
+            iconLocation="left"
+            onClick={() => history.push('/appointments/new')}
+          >
+            {t('patient.appointments.new')}
+          </Button>
```

Comment:
need to add some tests for this: 

- [ ] button gets rendered w/ correct label
- [ ] when the button gets clicked, it navigates to the correct location

Reply:
ok

Reply:
Btw can you please tell how to run tests on my local machine?

Reply:
`yarn test`